README.md

```diff
@@ -72,8 +72,10 @@ functionality:
    default)
 - `OS_<section>_\_<name>=<value>` - This format can be used to set arbitary
    Ironic config options
-- `IRONIC_RAMDISK_SSH_KEY` - A public key to allow ssh access to nodes running
-   IPA, takes the format "ssh-rsa AAAAB3....."
+- `IRONIC_RAMDISK_SSH_KEY` - A public key to allow ssh access as root to
+   nodes running IPA, takes the format "ssh-rsa AAAAB3.....". This relies on the
+   [dynamic-login](https://opendev.org/openstack/diskimage-builder/src/branch/master/diskimage_builder/elements/dynamic-login)
+   element to inject the key.
```

Comment:
Could you explicitly mention that only one key is accepted here? It's a common source of confusion.

Reply:
Better?